README.md

```diff
@@ -106,7 +106,7 @@ For any issues contact @nitesh on slack
   "error": 0,
   "data": 
   {
-      "id": 3,
+      "id": 3,conflicts
```

Comment:
in readme don't put the entire postman json. rather put the postman web link. that is better